packages/wrangler/src/index.tsx

```diff
@@ -666,8 +674,10 @@ export async function main(argv: string[]): Promise<void> {
         });
     },
     async (args) => {
-      const { filename } = args;
-      const config = args.config as Config;
+      const config = await readConfig(
+        (args.config as ConfigPath) ||
+          (args.script && (await findWranglerToml(path.dirname(args.script))))
```

Comment:
Do you need to run `findWranglerToml()` here, since that is exactly what `readConfig()` will do with the path.

Reply:
Hmm? It won't get the script passed to it, readConfig uses `process.cwd()` (which I can actually remove, since it's the default.)